05-deploying_shiny_app.Rmd

```diff
@@ -17,22 +17,20 @@ To create and deploy a Shiny app, you should complete the following steps:
 9.  [Add users to the app](#add-users).
 10. [Access the app](#access-the-app).
 
-### Copy the app template{#copy-template}
+### Use the app template{#use-template}
 
-To copy the app template:
+To create a new repo for the app, use the R Shiny app template:
 
-1.  Go to [github.com/new/import](https://github.com/new/import).
-2.  Fill in the form:
-    +   Your old repository’s clone URL: `https://github.com/moj-analytical-services/rshiny-template`
-    +   Owner: `moj-analytical-services`
-    +   Name: The name of your app, e.g., `my-app`
-    +   Privacy: Private
-6.  Select __Begin import__.
+1. Go to the [rshiny-template repo](https://github.com/moj-analytical-services/rshiny-template).
+2. Click __Use this template__.
```

Comment:
GDS guidance is to use select as opposed to click, as not all users may be using a mouse or trackpad.

```suggestion
2. Select __Use this template__.
```